### ISSUE TITLE
Accessibility - Department of Labor (DOL) Seal is Missing ALT Text 

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#16]
The The Department of Labor (DOL) seal is missing ALT text so it is reading the name of the file which is "DOLL png".

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting them to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for fixing the DOL seal ATLtext but there is now need to add that it is a logo when it is also a link.The LAT text should be "United States Department of Labor". Added the "Unites Sates" part so people will know what countries Department of Labor the application is for. 

Having the ALT text this way will help for the website SEO.

Will close this issue once fixed.

**Comment by adriennerlong** [comment]
John, is it sufficient to have the ALT text be "United States Department of Labor"? Or does it need to be "logo for the United States Department of Labor"?

**Comment by jfc3-dol** [comment]
@adriennerlong,

Do not need the word logo if it being used as a link to the Department of Labor website.

Plus, actually, the it's the Department of Labor seal not a logo. Other agencies have logos but Department of Labor has a seal. Only  reason I know this is because I worked on document that had to be accessible about how the seal and other agency logos had to be used and part of it was that the Department of Labor has a seal.

**Comment by adriennerlong** [comment]
Ok , thanks! 

ICF please change the ALT text to "United States Department of Labor"

**Comment by jfc3-dol** [comment]
@adriennerlong,

closing issues because the ALT text has been updated to "United States Department of Labor".
